unit/models/payment.py

```diff
@@ -111,6 +143,192 @@ def from_json_api(_id, _type, attributes, relationships):
+                           "description": description, "addenda": addenda, "status": status,
+                           "numberOfPayments": number_of_payments, "schedule": schedule, "tags": tags}
+        self.relationships = relationships
+
+
+class RecurringCreditAchPaymentDTO(BaseRecurringPaymentDTO):
+    @staticmethod
+    def from_json_api(_id, _type, attributes, relationships):
+        return RecurringCreditAchPaymentDTO(_id, _type, date_utils.to_datetime(attributes["createdAt"]),
+                                            date_utils.to_datetime(attributes["updatedAt"]), attributes["amount"],
+                                            attributes["description"], attributes.get("addenda"), attributes["status"],
+                                            attributes["numberOfPayments"],
+                                            Schedule.from_json_api(attributes["schedule"]), attributes.get("tags"),
+                                            relationships)
+
+
+class RecurringCreditBookPaymentDTO(BaseRecurringPaymentDTO):
+    @staticmethod
+    def from_json_api(_id, _type, attributes, relationships):
+        return RecurringCreditBookPaymentDTO(_id, _type, date_utils.to_datetime(attributes["createdAt"]),
+                                             date_utils.to_datetime(attributes["updatedAt"]), attributes["amount"],
+                                             attributes["description"], attributes.get("addenda"), attributes["status"],
+                                             attributes["numberOfPayments"],
+                                             Schedule.from_json_api(attributes["schedule"]), attributes.get("tags"),
+                                             relationships)
+
+
+class RecurringDebitAchPaymentDTO(BaseRecurringPaymentDTO):
+    @staticmethod
+    def from_json_api(_id, _type, attributes, relationships):
+        return RecurringDebitAchPaymentDTO(_id, _type, date_utils.to_datetime(attributes["createdAt"]),
+                                           date_utils.to_datetime(attributes["updatedAt"]), attributes["amount"],
+                                           attributes["description"], attributes.get("addenda"), attributes["status"],
+                                           attributes["numberOfPayments"],
+                                           Schedule.from_json_api(attributes["schedule"]), attributes.get("tags"),
+                                           relationships)
+
+
+RecurringPaymentDTO = Union[RecurringCreditAchPaymentDTO, RecurringDebitAchPaymentDTO, RecurringCreditBookPaymentDTO]
+
+
+class CreateRecurringCreditAchPaymentRequest(UnitRequest):
+    def __init__(self, amount: int, description: str, relationships: Dict[str, Relationship],
+                 idempotency_key: Optional[str], tags: Optional[Dict[str, str]], direction: str = "Credit",
+                 type: str = "achPayment"):
```

Comment:
type should be recurringCreditAchPayment? 
<img width="465" alt="image" src="https://user-images.githubusercontent.com/97832762/237038840-ba0d0961-2d82-4bd6-a666-fc987214f3a3.png">